src/hooks/useAllLiquidityMiningCampaigns.ts

```diff
@@ -82,7 +86,7 @@ const REGULAR_CAMPAIGN = gql`
   }
 `
 
-export function useAllLiquidtyMiningCampaigns(
+export function useAllLiquidityMiningCampaigns(
   pair?: Pair,
   dataFilter?: PairsFilterType
 ): {
```

Comment:
If you add `DisplayedMiningCampaign` interface I mentioned below you can rely on return type inference and get rid of 
```ts
: {
  loading: boolean
  miningCampaigns: any
}
```

This way rest of the app that uses this hook will be typed without `any`